docs/docs/Contributing/contributing-github-issues.md

```diff
@@ -4,21 +4,14 @@ sidebar_position: 2
 slug: /contributing-github-issues
 ---
 
+Our [issues](https://github.com/langflow-ai/langflow/issues) page is kept up to date with bugs, improvements, and feature requests. There is a taxonomy of labels to help with sorting and discovery of issues of interest. For an overview of the system we use to tag our issues and pull requests, see the Langflow repo's [labels page](https://github.com/langflow-ai/langflow/labels).
 
+## GitHub Discussions
```

Comment:
I'm not sure why this is included in information about contributing. I this this would be better in an FAQ under "How can I get help?

Reply:
Moved to a new page in Contributing